### PR TITLE
Make sure default timeouts are documented explicitly.

### PR DESCRIPTION
**Pull request by swankjesse**
Closes: https://github.com/square/okhttp/issues/4457

### CODE REVIEW

File: okhttp/src/main/java/okhttp3/OkHttpClient.java [hunk 1]
```diff
@@ -316,27 +316,27 @@ private static SSLSocketFactory newSslSocketFactory(X509TrustManager trustManage
...
   }
 
-  /** Default write timeout (in milliseconds). */
+  /** Default write timeout (in milliseconds). The default is 10 seconds. */
   public int writeTimeoutMillis() {
     return writeTimeout;
   }
 
-  /** Web socket ping interval (in milliseconds). */
+  /** Web socket and HTTP/2 ping interval (in milliseconds). By default no pings are sent. */
```
Comment (oldergod) [inline okhttp/src/main/java/okhttp3/OkHttpClient.java]: super nit but 'no ping is sent' ?

File: okhttp/src/main/java/okhttp3/OkHttpClient.java [hunk 2]
```diff
@@ -316,27 +316,27 @@ private static SSLSocketFactory newSslSocketFactory(X509TrustManager trustManage
     }
   }
 
-  /** Default call timeout (in milliseconds). */
+  /** Default call timeout (in milliseconds). By default calls have no timeout. */
```
Comment (JakeWharton) [inline okhttp/src/main/java/okhttp3/OkHttpClient.java]: This should probably clarify that individual timeouts may still apply

File: okhttp/src/main/java/okhttp3/OkHttpClient.java (hunk 1 above)
Comment (swankjesse) [reply okhttp/src/main/java/okhttp3/OkHttpClient.java]: _By default pings are not sent_

File: okhttp/src/main/java/okhttp3/OkHttpClient.java (hunk 2 above)
Comment (swankjesse) [reply okhttp/src/main/java/okhttp3/OkHttpClient.java]: _Default call timeout (in milliseconds). By default there is no timeout for complete calls, but there is for the connect, write, and read actions within a call._

**Pull request merged**